projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -297,19 +315,15 @@ function ( $payment_gateway ) {
 				$properties = $this->process_event_properties(
 					'woocommerceanalytics_product_checkout',
 					array(
-						'pq'                => $cart_item['quantity'],
-						'payment_options'   => $enabled_payment_options,
-						'device'            => wp_is_mobile() ? 'mobile' : 'desktop',
-						'guest_checkout'    => 'Yes' === $guest_checkout ? 'Yes' : 'No',
-						'create_account'    => 'Yes' === $create_account ? 'Yes' : 'No',
-						'express_checkout'  => 'null',
-						'shipping_option'   => $this->get_shipping_option_for_item( $cart_item_key ),
-						'products_count'    => $products_count,
-						'coupon_used'       => $coupon_used,
-						'order_value'       => $order_value,
-						'store_currency'    => get_woocommerce_currency(),
-						'additional_blocks' => $this->additional_blocks_on_page,
-						'template_used'     => $this->cart_checkout_templates_in_use ? '1' : '0',
+						'pq'               => $cart_item['quantity'],
+						'payment_options'  => $enabled_payment_options,
+						'guest_checkout'   => 'Yes' === $guest_checkout ? 'Y' : 'N',
+						'create_account'   => 'Yes' === $create_account ? 'Y' : 'N',
```

Comment:
I wonder what the benefit of using `Y` and `N` over `Yes` and `No` are? If we'd use `Yes` and `No`, we could simply use the following part:

```suggestion
						'guest_checkout'   => $guest_checkout,
						'create_account'   => '$create_account,
```

If we'd proceed with `Y` and `N`, we might want to adjust the corresponding part in `woocommerceanalytics_product_purchase` accordingly.